samples/web/content/apprtc/params.html

```diff
@@ -0,0 +1,97 @@
+
+    </section>
+
+    <section>
+
+      <table>
+        <tr><td><a href="https://apprtc.appspot.com?hd=true">hd=true</a></td><td>Use HD camera resolution <a href="http://googlechrome.github.io/webrtc/samples/web/content/getusermedia/resolution/" title="Video constraints demo">constraints</a>, i.e. minWidth: 1280, minHeight: 720</td></tr>
+        <tr><td><a href="https://apprtc.appspot.com?stereo=true">stereo=true</a></td><td>Turn on stereo audio</td></tr>
+        <tr><td><a href="https://apprtc.appspot.com?debug=loopback">debug=loopback</a></td><td>Connect to yourself, e.g. to test firewalls</td></tr>
+        <tr><td><a href="https://apprtc.appspot.com?ts=turn:foo">ts=[turnserver]</a></td><td>Set TURN server different from the default</td></tr>
+        <tr><td><a href="https://apprtc.appspot.com?tp=foo">tp=[password]</a></td><td>Set password for the TURN server</td></tr>
+        <tr><td><a href="https://apprtc.appspot.com?audio=true&amp;video=false">audio=true&amp;video=false</a></td><td>Audio only</td></tr>
+        <tr><td><a href="https://apprtc.appspot.com?audio=false">audio=false</a></td><td>Video only</td></tr>
+        <tr><td><a href="https://apprtc.appspot.com?audio=googEchoCancellation=false,googAutoGainControl=true">audio=googEchoCancellation=false,googAutoGainControl=true</a></td><td>Disable echo cancellation and enable gain control</td></tr>
+        <tr><td><a href="https://apprtc.appspot.com?audio=googNoiseReduction=true">audio=googNoiseReduction=true</a></td><td>Enable noise reduction</td></tr>
+        <tr><td><a href="https://apprtc.appspot.com?asc=ISAC/16000">asc=ISAC/16000</a></td><td>Set preferred audio send codec to be ISAC at 16kHz (use on Android)</td></tr>
+        <tr><td><a href="https://apprtc.appspot.com?arc=opus/48000">arc=opus/48000</a></td><td>Set preferred audio receive codec Opus at 48kHz</td></tr>
+        <tr><td><a href="https://apprtc.appspot.com?dtls=false">dtls=false</a></td><td>Disable <a href="https://en.wikipedia.org/wiki/Datagram_Transport_Layer_Security" title="Wikipedia article: DTLS">Datagram Transport Layer Security</a></td></tr>
+        <tr><td><a href="https://apprtc.appspot.com?dscp=true">dscp=true</a></td><td>Enable DSCP</td></tr>
+        <tr><td><a href="https://apprtc.appspot.com?ipv6=true">ipv6=true</a></td><td>Enable IPv6</td></tr>
+        <tr><td><a href="https://apprtc.appspot.com?arbr=48000">arbr=[bitrate]</a></td><td>Set audio receive bitrate, kbps</td></tr>
+        <tr><td><a href="https://apprtc.appspot.com?asbr=16000">asbr=[bitrate]</a></td><td>Set audio send bitrate</td></tr>
+        <tr><td><a href="https://apprtc.appspot.com?vsbr=1000">vsbr=[bitrate]</a></td><td>Set video receive bitrate</td></tr>
+        <tr><td><a href="https://apprtc.appspot.com?vrbr=8000">vrbr=[bitrate]</a></td><td>Set video send bitrate</td></tr>
```

Comment:
missing 'wshost', 'wsport', and 'wstls', for specifying the websocket server host, port, and if tls is used, respectively.